clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -61,14 +61,35 @@ type schedulerLatencyMeasurement struct{}
 // - reset - Resets latency data on api scheduler side.
 // - gather - Gathers and prints current scheduler latency data.
 func (s *schedulerLatencyMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
-	action, err := util.GetString(config.Params, "action")
+	isSSHSupported := config.ClusterFramework.GetClusterConfig().IsSSHSupported
+
+	c := config.ClusterFramework.GetClientSets().GetClient()
+	nodes, err := c.CoreV1().Nodes().List(metav1.ListOptions{})
```

Comment:
Hmm, this is actually a quite heavy call (especially in large clusters) we shouldn't be making it for every Execute call. I'm aware that it was already like this, but let's make it better. Would you mind moving this logic to clusterloader.go's `completeConfig` method and adding a field to the ClusterConfig? Ideally, it could be done in a separate PR. Thanks!

Reply:
I can tackle this in a separate PR, sure. I assume it's done here because we need to check if the master is registered. I don't love that i'm adding a bunch of new fields to this struct, but I could add something like `IsMasterNodeRegistered` to cover this and set it using this logic in `completeConfig`? WDYT?

Reply:
Yeah, that sounds great, thanks!

Reply:
Yes - let's do in a separate PR please.